### PR TITLE
New default ports

### LINKED ISSUE #11974: Change default ports for conflicting services
Currently Erigon has 3 (!) services using all the same port number by default (6060).

Problem is, this is very bad UX, by default diagnostics is on, if you turn metrics or pprof on, you also need to be aware of diagnostics using the same port and change it, otherwise you'll get surprised by something not working as expected.

We should not "surprise" users by giving different default port numbers for each service. I suggest keeping 6060 to pprof and changing the others.

Erigon 3 seems to be a good time to make this breaking change in the defaults.

```
~/erigon3/build/bin/erigon --help | grep 6060
   --pprof.port value                                                               pprof HTTP server listening port (default: 6060)
   --metrics.port value                                                             Metrics HTTP server listening port (default: 6060)
   --diagnostics.endpoint.port value                                                Diagnostics HTTP server listening port (default: 6060)
```

### PR DESCRIPTION
**Pull request by JkLondon**
closes #11974

### CODE REVIEW

**Comment by JkLondon** [comment]
@wmitsuda maybe I've missed some places in docs or somewhere else. Hope it's not but if you find just tag me here or smwr else

**Review by AskAlexSharov — APPROVED** [review]

**Comment by wmitsuda** [comment]
> @wmitsuda maybe I've missed some places in docs or somewhere else. Hope it's not but if you find just tag me here or smwr else

LGTM

**Comment by wmitsuda** [comment]
also tagging @bloxster FYI for E3 docs.

**Review by dvovk — CHANGES_REQUESTED** [review]
Please change ports :
https://github.com/erigontech/erigon/blob/fec56074c400025b27c2c232c19813cbd7c80115/cmd/diag/flags/flags.go#L29

https://github.com/erigontech/erigon/blob/fec56074c400025b27c2c232c19813cbd7c80115/turbo/app/support_cmd.go#L66

**Comment by JkLondon** [comment]
> Please change ports :
> 
> https://github.com/erigontech/erigon/blob/fec56074c400025b27c2c232c19813cbd7c80115/cmd/diag/flags/flags.go#L29
> 
> https://github.com/erigontech/erigon/blob/fec56074c400025b27c2c232c19813cbd7c80115/turbo/app/support_cmd.go#L66

I've changed them to 6063 (not used by anything else, as I know)

**Comment by dvovk** [comment]
@JkLondon  Sorry that I didn't provide more info, it is diagnostics so ports must be 6062

**Review by dvovk — APPROVED** [review]

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -659,7 +659,7 @@ var (
 	MetricsPortFlag = cli.IntFlag{
 		Name:  "metrics.port",
 		Usage: "Metrics HTTP server listening port",
-		Value: metrics.DefaultConfig.Port,
```
Comment (AskAlexSharov) [inline cmd/utils/flags.go:662]: then need remove `metrics.DefaultConfig.Port` field?

Comment (JkLondon) [reply cmd/utils/flags.go:662]: https://github.com/erigontech/erigon/pull/12111 @AskAlexSharov here it is.

**Pull request merged**